docs/ManualClusterSetup.md

```diff
@@ -0,0 +1,42 @@
+# Manual Cluster Setup
+
+This document describes requirements and best practices for setting up a cluster for trying the Calico Docker Prototype.
+
+## Requirements
+
+2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+
+They must have the following software installed.
+ * Docker v1.4 or greater [Installing Docker](https://docs.docker.com/installation/)
+ * etcd installed and available on each node. [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
```

Comment:
They also need a kernel of a certain version and some kernel modules loaded (iptables etc...)